### DISCUSSION TITLE
No Internet....

### DISCUSSION THREAD

**Discussion by macster2075** [AdguardTeam/AdGuardHome#3557] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Screenshots

<details><summary>Screenshot:</summary>

</details>

### Additional Information

running latest version. I have tried multiple times. I am not able to access the admin page and when I do and set it up, I lose internet as soon as I sent DNS IP address to point to Adguard.

**Comment by ameshkov** [comment]
To troubleshoot this issue we need to see AdGuard Home logs.

1. [Configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
2. Reproduce the issue.
3. Post the log file here.
